aws_embedded_metrics/validator.py

```diff
@@ -81,6 +84,14 @@ def validate_metric(name: str, value: float, unit: Optional[str]) -> None:
     if unit is not None and unit not in Unit:
         raise InvalidMetricError(f"Metric unit is not valid: {unit}")
 
+    if storageResolution is None or storageResolution not in StorageResolution:
+        raise InvalidMetricError(f"Metric storage Resolution is not valid: {storageResolution}")
+
+    if metricNameAndResolutionMap and name in metricNameAndResolutionMap:
+        if metricNameAndResolutionMap.get(name) is not storageResolution:
```

Comment:
```suggestion
        if if metricNameAndResolutionMap and name in metricNameAndResolutionMap and metricNameAndResolutionMap.get(name) is not storageResolution : 
```
Can't we club this into one?

Reply:
Done